scripts/run_go_codecov.sh

```
@@ -16,61 +16,62 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FILE_COVERAGE_INFO="go_coverage.txt"
-FILE_COVERAGE_HTML="go_coverage.html"
+FILE_COVERAGE_INFO="$PWD/go_coverage.txt"
+FILE_COVERAGE_HTML="$PWD/go_coverage.html"
```

Shellcheck (0.11.0)

[warning] FILE_COVERAGE_HTML appears unused. Verify use (or export if used externally).

(SC2034)